spec/spec_helper.rb

```diff
@@ -86,6 +86,9 @@
   config.include InjectSession, type: :feature
   config.include FeatureHelpers, type: :feature
   config.include EnvHelpers, type: :feature
+
+  # Time helpers
+  config.include ActiveSupport::Testing::TimeHelpers
```

Comment:
Do we actually use these in the end, or are they just necessary for the FactoryGirl stuff to work (I'm not sure why they would be). If we don't end up using them directly in our tests right now then we can avoid manually requiring them here (although they're likely loaded regardless).